book/src/fork-selection.md

```diff
@@ -0,0 +1,147 @@
+# Fork Selection
+This chapter defines Solana's *Nakomoto Fork Selection* algorithm based on time
+locks. It satisfies the following properties:
```

Comment:
Of these properties, which ones does Nakomoto Block Selection satisfy and which does this architecture add?

Reply:
```
* Rollback requires exponentially more time for older votes than for newer
  votes.
```
in PoW chain rollback requires exponentially more electricity to overcome the current chain.

Reply:
So maybe add a section that says how this algorithm compares to Nakomoto's? Or rephrase this to say it's "similar in spirit because like Nakomoto Block Selection, rollback requires exponentially more *something* for older *something*. Looks like a tricky comparison to me. Moving it into a separate section should minimize the wtfs-per-sec as someone familiar with Nakomoto Block Selection reads about time and votes.

Reply:
Or said another way, it'd be good to see an uncontroversial first sentence in this proposal. :)